apps/api/src/app/blueprint/e2e/consts.ts

```diff
@@ -0,0 +1,89 @@
+import { INotificationTemplate, StepTypeEnum } from '@novu/shared';
+
+const STEPS = [
+  {
+    _id: 'digest',
+    name: 'Digest',
+    template: {
+      type: StepTypeEnum.DIGEST,
+      content: '',
+    },
+  },
+  {
+    _id: 'email',
+    name: 'Email',
+    template: {
+      name: 'Test email',
+      subject: 'Test subject',
+      title: 'Test title',
+      type: StepTypeEnum.EMAIL,
+      content: 'adsf',
```

Comment:
```suggestion
      content: 'content',
```
To avoid spellcheck warn.

Reply:
Sorry this file is temporary, I am waiting for the actual data that will be created, then I will update it.